src/TraceEvent/TraceLog.cs

```diff
@@ -845,9 +845,9 @@ internal void RegisterStandardParsers()
             new HeapTraceProviderTraceEventParser(this);
             new MicrosoftWindowsKernelFileTraceEventParser(this);
             new IisTraceEventParser(this);
-
             new SampleProfilerTraceEventParser(this);
-#if false 
+            new InteropTraceEventParser(this);
```

Comment:
Remove this as well.  You don't need it.   This means that your change does not affect TraceEvent at all (just PerfView).  This is how I want it.